techniques/system/common/1.0/promises.st

```diff
@@ -278,15 +278,40 @@ bundle agent initialize_ncf()
 &if(!INITIAL)&
 bundle agent configure_rudder_reporting_system {
   methods:
-    rudder_reporting_syslog.!reports_disabled::
+    !rsyslog_disabled.!reports_disabled.policy_server::
+      "define rsyslog configuration for relays" usebundle => rsyslog_system_settings;
+
+    (rudder_reporting_syslog|!agent_capability_http).!reports_disabled::
       "configure_syslog"    usebundle => check_log_system;
       "configure_reporting" usebundle => check_rsyslog_version;
 
-    (rudder_reporting_syslog.reports_disabled)|rudder_reporting_https::
+    reports_disabled|(rudder_reporting_https.rsyslog_disabled)::
       "remove_reporting"    usebundle => remove_rudder_syslog_configuration;
 
-    rudder_reporting_https::
+    rudder_reporting_https.agent_capability_http::
```

Comment:
Isn't it http_reporting ?
See https://github.com/Normation/rudder-packages/blob/master/rudder-agent/SOURCES/Makefile

Reply:
it is